common/src/main/scala/net/psforever/packet/game/AvatarImplantMessage.scala

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2016 PSForever.net to present
+package net.psforever.packet.game
+
+import net.psforever.packet.{GamePacketOpcode, Marshallable, PacketHelpers, PlanetSideGamePacket}
+import scodec.Codec
+import scodec.codecs._
+
+/**
+  * An `Enumeration` of the available implants.
+  */
+object ImplantType extends Enumeration {
+  type Type = Value
+  val advanced_regen,
+      targeting,
+      audio_amplifier,
+      darklight_vision,
+      melee_booster,
+      personal_shield,
+      range_magnifier,
+      unknown7,
+      silent_run,
+      surge = Value
+
+  implicit val codec = PacketHelpers.createEnumerationCodec(this, uint4L)
+}
+
+/**
+  * Change the state of the implant.<br>
+  * Write better comments.
+  * <br>
+  * Implant:<br>
+  * `
```

Comment:
ScalaDocs knows how to accommodate the <br> tag correctly.  It also handles the backtick mark being on its own line.

I'm not certain that it likes the rest of the packet, though.  When i run the command, it raises complaint about everything, especially `scodec` stuff.